kratos/testing/test_case.h

```diff
@@ -24,6 +24,8 @@
 
 // Project includes
 #include "testing/tester.h"
+#include "includes/kernel.h"
```

Comment:
Left over includes.

Reply:
mmm
to be honest i think we should include the kernel in the tests. The kernel should be initialized at least...